Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-commander/user-guide/sensitive-inputs.adoc

= Lisk Commander Sensitive Inputs
Mona Bärenfänger <mona@lightcurve.io>

> :description: The sensitive Inputs page describes the different ways how to provide sensitive data like passphrases to Lisk Commander.

Reviewer: ```suggestion
:description: The sensitive Inputs page describes the different methods of how to provide sensitive data such as passphrases to the Lisk Commander.
```
